### ISSUE TITLE
one-liner installation does not work

### ISSUE THREAD

**Issue by DD4WH** [UncleSamulus/BirdNET-stream#3]
* I installed the latest 64-bit Rpi OS (desktop version) with preset WiFi access on an SD card using Raspberry Pi Imager
* I connected to the RPi using putty from my PC via the local WiFi network
* I issued the one-liner install command `curl -sL https://raw.githubusercontent.com/UncleSamulus/BirdNET-stream/main/install.sh | bash`
* nothing happens, the cursor returns after one second
* no output, no function (browser on the PC does not find birdnet.home)
* when I issue `DEBUG=1 ./install.sh`, the Pi throws an error `no such file or directory`

What am I missing here ?

Sorry for this obviously simple question, but Linux is not my native language.

**Comment by samuelortion** [comment]
Sorry to hear that...
The tool is not very mature yet, and may not work properly.

However, on thing you could try is starting to download the install.sh script first on your raspberry pi:
```bash
 curl -O https://raw.githubusercontent.com/UncleSamulus/BirdNET-stream/dev/install.sh
```

This way, you will be able to run `DEBUG=1 BRANCH=dev ./install.sh`; after allowing file execution:
```bash
chmod +x ./install.sh
DEBUG=1 BRANCH=dev ./install.sh
```
Otherwise, using the curl command you ran, the installation file isn't saved as a file, and thus will not be available later for execution.

I am very interested in your tests on raspberry pi; I hope it will work without too much trouble...
Good luck and let me know !

**Comment by DD4WH** [comment]
Thanks a lot for your quick answer and your help!
Yes, that worked, but it seems there is something weird with the structure of your github branches:
![grafik](https://user-images.githubusercontent.com/14326464/186147804-1710e36a-f27a-4a49-a3cc-194fefc0e6b1.png)

**Comment by samuelortion** [comment]
Indeed, that's an embarassing typo. There should be a space between `$BRANCH` and --recurse-submodule.

I have pushed a modified version (I did findd this issue before, but not updated github).

You may need to wait a bit for github to update the repo.

I hope it will work better now.

I am currently running the installation script in a fresh debian vm, to check that out.

If there is any problem, do not hesitate to ask me some questions, or bug fix for the script.

**Comment by samuelortion** [comment]
Unfortunatly, there are still some issues. 
For instance, the webapp does not catch sql exceptions when there are no sql database available.

You may also need to change the nginx configuration a little in `/etc/nginx/sites-available/birdnet-stream.conf'. Generate a new ssl certificate, and more things that are not handled by the script for the moment.

**Comment by DD4WH** [comment]
Ok, thanks a lot for your quick and helpful feedback!
I am sorry, but the procedure you describe is far beyond my expertise.

**Comment by samuelortion** [comment]
There are a little explanations inside INSTALL.md, but It may not be as straightforward as BirdNET-Pi installation for the moment.

ps. You should not run the installation script inside ~/BirdNET-Pi, as It will create a directory for BirdNET-stream inside the BirdNET-Pi folder, and it may do bad stuff with it.
